CHANGELOG.md

```diff
@@ -1,3 +1,61 @@
+v1.10.0 / 2020-05-15
```

Comment:
move the contents under unreleased to 1.10.0 folder. Like this: https://github.com/openebs/zfs-localpv/pull/107

Reply:
done